frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -504,12 +505,19 @@
    "fieldname": "disable_user_pass_login",
    "fieldtype": "Check",
    "label": "Disable Username/Password Login"
+  },
+  {
+   "default": "0",
```

Comment:
Let's default to 1? Or maybe we can set it to 1 on completing of setup wizard.

Reply:
Lets leave it off by default for now.